### PR TITLE
One Page App

### PR DESCRIPTION
**Pull request by CRClothier**
In this checkpoint, the app was expanded into a one page app with three 'pages'.  The pages are loaded using react router for the 'home', 'calculator' and 'quote' pages.

### CODE REVIEW

File: src/App.js [hunk 1]
```diff
@@ -1,13 +1,30 @@
 /* eslint-disable react/prefer-stateless-function */
```
Comment (rloterh) [inline src/App.js]: - [x] You have **disabled linter checks** on your code. Please enable **all** linter check on your code to ensure that your code is robust code and clean.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Chris, 

You did extremely great on this project! 👏 👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦 🚦 🚦  

## Good Points 👍  
- Descriptive Pull Request
- Pages correctly routed using _React Router_
- Pages navigation is well implemented
- App is running as intended
- Nice UI
- No linter errors


## Required Changes ♻️
_Kindly check the comments under the review._


<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 2 more limited reviews per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/components/HomePage.js [hunk 2]
```diff
@@ -0,0 +1,34 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (rloterh) [inline src/components/HomePage.js]: - [x] Kindly enable **all** disabled linter checks in your code to enhance a cleaner and robust code, as requested in the previous review.

File: src/components/QuotePage.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (rloterh) [inline src/components/QuotePage.js]: - [x] Kindly enable **all** disabled linter checks in your code to enhance a cleaner and robust code, as requested in the previous review.

File: src/components/CalculatorPage.js [hunk 4]
```diff
@@ -0,0 +1,16 @@
+/* eslint-disable react/prefer-stateless-function */
```
Comment (rloterh) [inline src/components/CalculatorPage.js]: - [x] Kindly enable **all** disabled linter checks in your code to enhance a cleaner and robust code, as requested in the previous review.

**Review by rloterh — CHANGES_REQUESTED** [review]
Hi Chris, 

You did great on this project, and for making the changes requested in the previous review(s)! 👏👏 
There are however a few issues that you still need to work on to go to the next project but you are almost there! 🚦  🚦  

## Good Points 👍  
- No linter errors

## Required Changes ♻️
_Kindly check the comments below this review. Please take your time to carefully improve aspects outlined in the previous code review(s), and kindly check to ensure that the changes implemented are working as intended._

<br>
Cheers and Happy coding!👏 💻 🍷

Feel free to leave any questions or comments in the PR thread if something is not 100% clear. Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have 1 more limited review per this project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MahmoudBakr23 — APPROVED** [review]
Hi @CRClothier 😃, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:
Congratulations! 🎉 

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**